locale/en/download/package-manager.md

```diff
@@ -72,10 +73,20 @@ dnf module install nodejs:12
 
 For CentOS/RHEL 7 Node.js is available via [Software Collections](https://www.softwarecollections.org/en/scls/?search=NodeJS).
 
-## Debian and Ubuntu based Linux distributions, Enterprise Linux/Fedora and Snap packages
+### Alternatives
+
+Packages compatible with CentOS, Fedora and RHEL are available via
+* [Node.js snaps](#snap).
+* [NodeSource Node.js binary distributions](#debian-and-ubuntu-based-linux-distributions).
```

Comment:
We're not great about doing this, but as a rule, we should avoid lists that complete a sentence fragment that introduces the list because these can be difficult to translate to some languages.

```suggestion
These resources provide packages compatible with CentOS, Fedora, and RHEL.

* [Node.js snaps](#snap)
* [NodeSource Node.js binary distributions](#debian-and-ubuntu-based-linux-distributions)
```